deposit/src/main/java/edu/unc/lib/deposit/fcrepo4/IngestContentObjectsJob.java

```diff
@@ -630,6 +630,28 @@ private void ingestWork(ContentContainerObject parent, Resource parentResc, Reso
 
                 // Cease refreshing the transaction
                 txRefresher.stop();
+            } catch (ChecksumMismatchException e) {
```

Comment:
what about overriding the method signature to `private void ingestWork(ContentContainerObject parent, Resource parentResc, Resource childResc, int retries)` and the same thing without `retries`. Most stuff would call the version without retries, which would supply the starting number of retries at 3.

Then you should be able to do something like:
```
} catch (ChecksumMismatchException e) {
     // Rollback changes for the work so far
     txRefresher.interrupt();
     tx.cancel();
     if (retries > 0) {
         retries--;
         ingestWork(parent, parentResc, childResc, retries);
     } else {
         failJob(...);
     }
```
I think if a work had more than one file and failed partway through, it would try to reingest those other files and run into a conflict since they'd already exist.